### ISSUE TITLE
Loss Function Adjustment Recommendation due to valid loss = 0

### ISSUE THREAD

**Issue by rezachu** [DAMO-DI-ML/KDD2023-DCdetector#29]
The [previous issue card](https://github.com/DAMO-DI-ML/KDD2023-DCdetector/issues/5) concluded that the model did not train toward the loss. I have been tuning the loss function and not getting a good result. 
Wonder is there any recommendation for tuning the loss function setup. Many Thanks.

**Comment by yyysjz1997** [comment]
> The [previous issue card](https://github.com/DAMO-DI-ML/KDD2023-DCdetector/issues/5) concluded that the model did not train toward the loss. I have been tuning the loss function and not getting a good result. Wonder is there any recommendation for tuning the loss function setup. Many Thanks.

Hi, I don't know if you are referring to the adjustment of the loss function with a plus or minus sign or? Theoretically, our loss function doesn't need to be adjusted, you can preferably try to change the [hyperparameters ](https://github.com/DAMO-DI-ML/KDD2023-DCdetector/tree/main/scripts)for model optimization.
